### PR TITLE
Styling fixes for blade title buttons

### PR DESCRIPTION
**Pull request by redhog**
Connects https://github.com/SkyTruth/pelagos-server/issues/962
Connects https://github.com/SkyTruth/pelagos-server/issues/961

### CODE REVIEW

**Comment by enriquetuya** [comment]
@redhog I don't think this address the width issue detailed on https://github.com/SkyTruth/pelagos-server/issues/962 about the widht of the sidebar.

File: js/app/Visualization/UI/SidePanels/InfoUI.js [hunk 1]
```diff
@@ -15,17 +15,18 @@ define([
...
-                    '  <h2 data-dojo-attach-point="titleNode">Vessel Information</h2>' +
-                    '  <div class="loading-vessel-info" style="display: none;" data-dojo-attach-point="loadingNode">' +
-                         '<img style="width: 20px;" src="${app.dirs.loader}">'+
-                      '</div>' +
-                    '  <div id="vessel_identifiers" class="${baseClass}Container" data-dojo-attach-point="containerNode"></div>' +
-                    '</div>',
+    templateString: '' +
+      '<div class="${baseClass}" style="overflow: auto;">' +
+      '  <div class="titleButtons">' +
+      '    <a id="activate_search" class="activate_search" href="javascript:undefined" data-dojo-attach-event="click:activateSearch"><i class="fa fa-search"></i></a>' +
```
Comment (enriquetuya) [inline js/app/Visualization/UI/SidePanels/InfoUI.js:21]: Shouldn't we remove this line? we are including the search action button on the tile.

Comment (enriquetuya) [inline js/app/Visualization/UI/SidePanels/InfoUI.js:21]: Remove the activate_search from the css file, we are not using anymore.

Comment (redhog) [inline js/app/Visualization/UI/SidePanels/InfoUI.js:21]: Done, see below: https://github.com/SkyTruth/pelagos-client/pull/140/files#diff-52e61d11d683ca62097a13f72cf34713L682

Comment (redhog) [inline js/app/Visualization/UI/SidePanels/InfoUI.js:21]: Remove which line? The download kml? Then we'd break all kml löink code. This is only disabled until you/Paul decides you need the functionality again (which I'm sure will happen - I've removed and added this link many times now :)

Comment (enriquetuya) [inline js/app/Visualization/UI/SidePanels/InfoUI.js:21]: @redhog remove the 
`<a id="activate_search" class="activate_search" href="javascript:undefined" data-dojo-attach-event="click:activateSearch"><i class="fa fa-search"></i></a>`
not the download kml.

Comment (redhog) [inline js/app/Visualization/UI/SidePanels/InfoUI.js:21]: Then you couldn't search? This is the search button. The <div class="titleButtons"> moves it to the title bar.

Comment (enriquetuya) [inline js/app/Visualization/UI/SidePanels/InfoUI.js:21]: Oh, I see what you did. Understood.

**Pull request merged**